File: Selection_Sort/selection_sort.py
arr = [2,6,1,8,3]
n = 5


for i in range(n-1):
    for j in range(i, n):
        if (arr[i] > arr[j]):
            arr[i], arr[j] = arr[j], arr[i]


# for i in range(n-1):
#     min = i
#     for j in range(i, n):
#         if (arr[i] > arr[j]):
#             min = j
#     arr[i], arr[min] = arr[min], arr[i]

print(arr)

File: Selection_Sort/selection_sort.cpp
#include<bits/stdc++.h>

using namespace std;

int main(){
    int n;
    cin >> n;

    int arr[n];

    for(int i =0; i < n; i++){
        cin >> arr[i];
    }

    for(int i = 0; i <= n-2; i++){
        for(int j = i; j <= n-1; j++){
            if(arr[i] < arr[j]){
               swap(arr[i], arr[j]);
            }
        }
        
    }

    // for(int i = 0; i <= n-2; i++){
    //     int min = i;
    //     for(int j = i; j <= n-1; j++){
    //         if(arr[i] < arr[j]){
    //             min = j;
                
    //         }
    //     }
    //     swap(arr[i], arr[min]);
    // }


    for(int i =0; i < n; i++){
        cout << arr[i] << " ";
    }

    return 0;
}